utils/python/CIME/SystemTests/test_utils/user_nl_utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module contains functions for working with user_nl files in system tests.
```

Comment:
These methods look like they might be generally useful. Why did you decide to make a new directory instead of just adding a test_utils.py file to the common CIME area?